imap_processing/lo/packet_definitions/P_ILO_MEMDMP.xml

```diff
@@ -28,7 +28,10 @@
 				<xtce:UnitSet />
 				<xtce:BinaryDataEncoding bitOrder="mostSignificantBitFirst">
 					<xtce:SizeInBits>
-						<xtce:FixedValue>32608</xtce:FixedValue>
+						<xtce:DynamicValue>
+							<xtce:ParameterInstanceRef parameterRef="PKT_LEN"/>
+              				<xtce:LinearAdjustment intercept="-160" slope="8"/>
```

Comment:
I think negative values are good to go, but this might be an issue when parsing.

Reply:
The intercept is supposed to be negative here. It shouldn't be too bad to add to the generator. For the linear adjustment, the slope is always 8 and the intercept is -<total non-variable length bits in the packet>. There would just need to be some indication which packets/field is the dynamic value.